packages/donations-app/app/components/project-wizard/projectWizardOutcomeView.html

```diff
@@ -9,8 +9,24 @@
 </div>
 
 <!-- Visible when outcome is selected -->
+
+
+
 <div class="col-md-8" ng-repeat="outcome in prWizardCtrl.project.outcomes track by $index"
      ng-show="prWizardCtrl.mode == prWizardCtrl.modes.OUTCOMES && prWizardCtrl.selectedOutcome === outcome">
+
+  <div class="form-group">
+    <label class="col-sm-3 control-label">Outcome order number *</label>
+
+    <div class="col-sm-9">
+      <input type="text" class="form-control" ng-model="outcome.orderNumber"
```

Comment:
We could prepopulate it with a default being the current index in the outcome collection, so it's easier to use, especially for older projects. However, we can do it later - while fixing the wizard validations.

Reply:
We also can just set values manually for older projects